packages/controller/lib/setup.js

```diff
@@ -783,25 +771,39 @@ async function processCommand(command, args, params, callback) {
                                 adapters.push(objs.rows[i].value.common.name);
                             }
 
-                            upload.uploadAdapterFull(adapters, callback);
-                        });
+                            await upload.uploadAdapterFullAsync(adapters);
+                            callback();
+                        } catch (err) {
+                            console.error(`Cannot upload all adapters: ${err}`);
```

Comment:
`e.message` here and for all others in setup?

Reply:
All e => err